.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - "3.5"
 
 install:
-  - touch oclint
+  - echo '#!/bin/sh' > ./oclint
```

Comment:
Like the `echo` part. Why you prefixed `./`to `oclint` for this line but not the next? I think it's safe to just leave it as `oclint`.